Neos.Flow/Configuration/Objects.yaml

```diff
@@ -367,6 +367,10 @@ Doctrine\ORM\EntityManagerInterface:
   scope: singleton
   factoryObjectName: Neos\Flow\Persistence\Doctrine\EntityManagerFactory
 
+Doctrine\DBAL\Connection:
+  scope: singleton
```

Comment:
might that pose a problem if you work with multiple different connections?

Reply:
The keyword being "if" :-D So, should the general injection be the Flow configured and then we have a "createWithOptions" static method (which can then be used in entityMangerFactory?) if you want a diffrent connection, following Doctrines configuraton format?

Reply:
No, that's not what I meant.
I just wonder if this will break if you use a custom factory for your own connection instances

Reply:
Arh, okay :) But then you would set it as a property in your own class, and register your own factory. so the question here would be, which one (this registration or your own factory) will take care of creatig the instance?

Reply:
i.e. will this work:
```php
class SomeClass {

  /**
   * @Flow\Inject
   * @var Connection
   */
  protected $flowConnection;
 
  /**
   * @Flow\Inject
   * @var Connection
   */
  protected $customConnection;
}
```

..if you configure a custom factory for the 2nd connection like

```yaml
Your\Package\SomeClass:
  properties:
    customConnection:
      object:
        factoryObjectName: Your\Package\SomeCustomConnectionFactory
```

or will the two instances be the same since its a singleton now?

Reply:
I will test that 👍

Reply:
Tested with positive result - see comment here

https://github.com/neos/flow-development-collection/pull/2169#issuecomment-708324572